src/iotics/host/api/follow_api.py

```diff
@@ -83,16 +83,21 @@ def on_message(self, headers: dict, body):
             self._message_handler(headers, deserialised_resp)
 
     def on_error(self, headers: dict, body):
-        logger.error('Received stomp error body: %s headers: %s', body, headers)
+        level = logging.ERROR
         try:
             # This will be improved once https://ioticlabs.atlassian.net/browse/FO-1889 will be done
             error = get_stomp_error_message(body) or 'No error body'
```

Comment:
Seems like good idea to do this improvement while we're here?
i.e. check for code == 16
```{"code":16,"message":"UNAUTHENTICATED: token expired"}```